src/ignitus-Shared/ignitus-DesignSystem/ignitus-Atoms/ignitus-defaultTag/styles.ts

```diff
@@ -0,0 +1,62 @@
+import styled from '@emotion/styled';
+import Icon from '../../../ignitus-Utilities/Components/icon';
+
+export const StyledIcon = styled(Icon)`
+  height: 1.2rem;
+  width: 1.2rem;
+  fill: black;
+  transition: 0.3s;
+  flex: 1;
+`;
+
+export const StyledTag1 = {
+  backgroundColor: "#FFA500",
+  color: "white",
+  fontWeight: 600,
+  textAlign: "center",
+  borderRadius: "45px",
+  margin: "0.5rem",
+  lineHeight: "1rem",
+  fontSize: "18px",
+  padding: "16px 32px",
+  display: "-webkit-inline-box"
+} as React.CSSProperties;
+
+export const StyledTag2 = {
+  backgroundColor: "#A9A9A9",
+  color: "#000066",
+  fontWeight: 600,
+  textAlign: "center",
+  borderRadius: "45px",
+  margin: "0.5rem",
+  lineHeight: "1rem",
+  fontSize: "18px",
+  padding: "16px 32px",
+  display: "-webkit-inline-box"
+} as React.CSSProperties;
+
+export const StyledTag3 = {
+  backgroundColor: "#000066",
+  color: "#ffffff",
+  fontWeight: 600,
+  textAlign: "center",
+  borderRadius: "45px",
```

Comment:
wither use `rem` or px, I would say `rem` as we use it in rest of the app.

Reply:
Done